### DISCUSSION TITLE
Where to import

### DISCUSSION THREAD

**Discussion by idc77** [fontsource/fontsource#1003] [Q&A]
> In your application’s entry file, page, or site component, import the font package using the package name. For example, to import Open Sans into index.jsx, add the following code:

Using React vite, where do I import it?
in main.tsx or index.css?

**Comment by jwr1** [comment]
`main.tsx` should be fine.

All it's saying is that it should be imported somewhere that would keep it available to the whole application. For example, if you just imported it in a specific component file, then you might get some pages that have the font loaded and some that don't, depending on how your framework handles imports, but if you import fontsource in your entry point (`index.tsx`, `main.tsx`, `App.tsx`, or whatever you called it), then the fonts will be available to the whole application.
